### PR TITLE
Describe the shift visualRangeUpdateMode for the chart valueAxis

### PR DESCRIPTION
**Pull request by AlexJameson**
https://supportcenter.devexpress.com/internal/ticket/details/T856912

### CODE REVIEW

File: api-reference/20 Data Visualization Widgets/dxChart/1 Configuration/valueAxis/visualRangeUpdateMode.md [hunk 1]
```diff
@@ -19,8 +19,11 @@ The visual range does not change.
 
 - *"auto"*      
 When the visual range is specified, the applied mode changes depending on the argument axis' [visualRangeUpdateMode](/api-reference/20%20Data%20Visualization%20Widgets/dxChart/1%20Configuration/argumentAxis/visualRangeUpdateMode.md '/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/argumentAxis/#visualRangeUpdateMode') at the moment when data is updated:
-    - if *"reset"* or *"shift"*, then *"reset"*;
+    - if *"reset"*, then *"reset"*;
     - if *"keep"*, then *"keep"*.
 
+- *"Shift"*
```
Comment (RomanTsukanov) [inline api-reference/20 Data Visualization Widgets/dxChart/1 Configuration/valueAxis/visualRangeUpdateMode.md]:
```suggestion
- *"shift"*     
```

File: api-reference/20 Data Visualization Widgets/dxChart/1 Configuration/valueAxis/visualRangeUpdateMode.md [hunk 2]
```diff
@@ -19,8 +19,11 @@ The visual range does not change.
 
 - *"auto"*      
 When the visual range is specified, the applied mode changes depending on the argument axis' [visualRangeUpdateMode](/api-reference/20%20Data%20Visualization%20Widgets/dxChart/1%20Configuration/argumentAxis/visualRangeUpdateMode.md '/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/argumentAxis/#visualRangeUpdateMode') at the moment when data is updated:
-    - if *"reset"* or *"shift"*, then *"reset"*;
+    - if *"reset"*, then *"reset"*;
     - if *"keep"*, then *"keep"*.
```
Comment (RomanTsukanov) [inline api-reference/20 Data Visualization Widgets/dxChart/1 Configuration/valueAxis/visualRangeUpdateMode.md]: This means that the value axis' mode is the same as the argument axis' mode. These ifs are unnecessary.

**Review by RomanTsukanov — CHANGES_REQUESTED** [review]

**Review by alexanderPolosatov — APPROVED** [review]
LGTM

**Review by RomanTsukanov — APPROVED** [review]

File: api-reference/20 Data Visualization Widgets/dxChart/1 Configuration/valueAxis/visualRangeUpdateMode.md [hunk 3]
```diff
@@ -18,9 +18,10 @@ The visual range becomes equal to the data range or the [whole range](/api-refer
 The visual range does not change.
 
 - *"auto"*      
-When the visual range is specified, the applied mode changes depending on the argument axis' [visualRangeUpdateMode](/api-reference/20%20Data%20Visualization%20Widgets/dxChart/1%20Configuration/argumentAxis/visualRangeUpdateMode.md '/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/argumentAxis/#visualRangeUpdateMode') at the moment when data is updated:
-    - if *"reset"* or *"shift"*, then *"reset"*;
-    - if *"keep"*, then *"keep"*.
+When the visual range is specified, the applied mode is the same as the argument axis' [visualRangeUpdateMode](/api-reference/20%20Data%20Visualization%20Widgets/dxChart/1%20Configuration/argumentAxis/visualRangeUpdateMode.md '/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/argumentAxis/#visualRangeUpdateMode').
+
+- *"shift"*     
+The same as the *"auto"* mode.
```
Comment (dirk-pieterse) [inline api-reference/20 Data Visualization Widgets/dxChart/1 Configuration/valueAxis/visualRangeUpdateMode.md]:
```suggestion
The same as *"auto"* mode.
```

**Review by dirk-pieterse — APPROVED** [review]

**Pull request merged**